README.md

```diff
@@ -63,7 +63,7 @@ With the same example as above, do the following for each repository beforehand
     cd repo
 
     git filter-branch --index-filter \
-      'tab=$(printf "\t") && git ls-files -s | sed "s-$tab\"*-&newsubdir/-" | GIT_INDEX_FILE=$GIT_INDEX_FILE.new git update-index --index-info && mv "$GIT_INDEX_FILE.new" "$GIT_INDEX_FILE"' \
+      'tab=$(printf "\t") && git ls-files -s | sed "s-$tab\"*-&newsubdir/-" | GIT_INDEX_FILE=$GIT_INDEX_FILE.new git update-index --index-info && mv "$GIT_INDEX_FILE.new" "$GIT_INDEX_FILE" 2>/dev/null || true' \
```

Comment:
I'd check the result of `git ls-files -s` to skip the following commands when the index is empty, rather than hide and ignore any error.
Something like: `'tab=$(printf "\t") && git ls-files -s --error-unmatch . >/dev/null 2>&1 ; [ $? != 0 ] || git ls-files -s | sed "s-$tab\"*-&newsubdir/-" | GIT_INDEX_FILE=$GIT_INDEX_FILE.new git update-index --index-info && mv "$GIT_INDEX_FILE.new" "$GIT_INDEX_FILE"'`